### PR TITLE
Use Governable contract from random-beacon module

### PR DESCRIPTION
**Pull request by pdyraga**
~~Depends on #2945; keeping it as a draft until #2945 is not merged~~

To do not duplicate exactly the same code, `Governable` is declared next
to `Reimbursable` in `random-beacon` module and we now import it from
`ecdsa` module.

~~There is a temporary workaround in `package.json` to import from
`../random-beacon` instead of the development tag. This will have to be
changed back once #2945 moving files to `random-beacon` gets merged and a
new package gets published. See commit d99ab6057242c156d07784147387b973a9cb8fb8.~~

### CODE REVIEW

File: solidity/ecdsa/package.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
-    "@keep-network/random-beacon": "development",
+    "@keep-network/random-beacon": "../random-beacon",
```
Comment (nkuba) [inline solidity/ecdsa/package.json:65]: Want to leave it as is or set it back to `development` once an updated package is published?

Comment (pdyraga) [reply solidity/ecdsa/package.json:65]: Waiting for a new package to be published, I will switch it in this PR.

Comment (pdyraga) [reply solidity/ecdsa/package.json]: Switched back.

**Comment by pdyraga** [comment]
Undrafted @nkuba

**Review by nkuba — APPROVED** [review]

**Pull request merged**